docs/Troubleshooting.md

```diff
@@ -97,3 +97,57 @@ Again, there are a number of possible causes and solutions:
   been seen recently. Please, post in issue
   [#53](https://github.com/nteract/hydrogen/issues/53) the details of your
   installation.
+
+# Hydrogen No kernel specs found Issue!
```

Comment:
Since you could get a `No kernel specs found` message for a number of reasons, what do you think about changing this to:

```
# Setting environment variables on Windows
```
or something to that effect?

Reply:
That's a great suggestion.
I will update the title 👍